vendor/assets/stylesheets/components/_chips.scss

```diff
@@ -0,0 +1,27 @@
+.chip {
+  display: inline-block;
+  height: 32px;
+  font-size: 13px;
+  font-weight: 500;
+  color: rgba(0,0,0,.6);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.6)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`.6` should be written with a leading zero as `0.6`<br>Commas in function arguments should be followed by a single space